src/main/java/com/acrolinx/client/demo/SdkDemo.java

```diff
@@ -119,4 +122,50 @@ private static AccessToken signInInteractive(AcrolinxEndpoint acrolinxEndpoint)
             urlString -> LOGGER.info("Please sign in at: {}", urlString));
     return signInSuccess.getAccessToken();
   }
+
+  private static void checkSimpleText()
```

Comment:
The `checkSimpleText()` and `checkDocxFile()` methods are very similar/redundant. We can improve this by only having a single method which takes the `CheckRequest` as a parameter.

Reply:
done